crystal-map-connector/src/main/kotlin/ch/ergon/dope/extension/type/arithmetic/NumberInfix.kt

```diff
@@ -51,6 +61,9 @@ fun Number.div(number: CMField<out Number>) = toDopeType().div(number.toDopeType
 
 fun CMField<out Number>.div(number: Number) = toDopeType().div(number.toDopeType())
 
+fun <KotlinType : Any, MapType : Number> KotlinType.div(other: CMConverterField<KotlinType, MapType>) =
```

Comment:
Making the upper bound of _KotlinType_ _Any_ is just wrong. _Div_ is only defined on numbers

Reply:
Also, I don't really get why you decide for _KotlinType_ and _MapType_? Can't we name it _FieldType_ and _ConvertedFieldType_ or something?

Reply:
No, The Any isn't wrong. It needs to be any. The only thing that needs to be a Number is the MapType, which is one.
I coppied the naming from Crystal-Map, but i could change it if you think that it is really necessary.

Reply:
maybe name it _Convertable_ and JSONType/_JsonType_

Reply:
done